network-api/networkapi/wagtailpages/templates/fragments/research_details.html

```diff
@@ -0,0 +1,23 @@
+{% load i18n localization %}
+
+{% comment %}
+ This is the section underneath the title of the products heading, where we display date and time spent researching, as well as creepiness level people have voted.
+{% endcomment %}
+
+<div class="col-12 mt-3 tw-flex xsmall:tw-flex-col medium:tw-flex-wrap large:tw-flex-row">
```

Comment:
Hi @fessehaye! Tagged you in this PR as I was hoping you could take a look at my use of tailwind for responsiveness and let me know what you think. Thanks!